[    0.00] [  OK  ] LOW_MEM mode disabled
[    0.00] [ DEBUG] Hell debug
[    0.00] [  OK  ] Loading package managment...
[    0.05] [  OK  ] Mounting system drive
[    0.10] [  OK  ] Mounting table: 0x7f4e8013f480 to /
[    0.10] [  OK  ] Initializing registry
[    0.20] [  OK  ] Registry: Mounted /PlotOS/system32/registry/system.reg to system
[    0.20] [ DEBUG] Registry: trying to lock /PlotOS/system32/registry/system.reg r
[    0.20] [ DEBUG] Registry: locking /PlotOS/system32/registry/system.reg r
[    0.20] [  OK  ] Registry: reading /PlotOS/system32/registry/system.reg
[    0.20] [  OK  ] Opening file at /PlotOS/system32/registry/system.reg with mode rb
[    0.25] [  OK  ] Registry: Mounted /PlotOS/system32/registry/user_root.reg to user_root
[    0.25] [ DEBUG] Registry: trying to lock /PlotOS/system32/registry/user_root.reg r
[    0.25] [ DEBUG] Registry: locking /PlotOS/system32/registry/user_root.reg r
[    0.25] [  OK  ] Registry: reading /PlotOS/system32/registry/user_root.reg
[    0.25] [  OK  ] Opening file at /PlotOS/system32/registry/user_root.reg with mode rb
[    0.25] [ DEBUG] registry.get("system/registry/use_tmp_files_to_save, nil") called
[    0.25] [  OK  ] Registry use tmp files to save enabled
[    0.25] [ DEBUG] registry.get("system/low_mem, nil") called
[    0.25] [  OK  ] Loading drivers...
[    0.35] [FAILED] Error loading file /lib/process.lua
[    0.35] [FAILED] Error: /lib/process.lua:2: attempt to call a nil value (global 'require')
[    0.35] [FAILED] stack traceback:
[    0.35] [FAILED]     [C]: in function 'xpcall'
[    0.35] [FAILED]     machine:823: in upvalue 'errorCapture'
[    0.35] [FAILED]     machine:827: in global 'require'
[    0.35] [FAILED]     /lib/process.lua:2: in main chunk
[    0.35] [FAILED]     [C]: in function 'xpcall'
[    0.35] [FAILED]     machine:823: in local 'errorCapture'
[    0.40] [FAILED]     machine:833: in global 'xpcall'
[    0.40] [FAILED]     /PlotOS/bootInit.lua:241: in global 'raw_dofile'
[    0.40] [FAILED]     /lib/package.lua:11: in field 'require'
[    0.40] [FAILED]     /lib/ipc.lua:1: in main chunk
[    0.40] [FAILED]     ...
[    0.40] [FAILED]     machine:823: in local 'errorCapture'
[    0.40] [FAILED]     machine:833: in global 'xpcall'
[    0.40] [FAILED]     /PlotOS/bootInit.lua:241: in global 'raw_dofile'
[    0.40] [FAILED]     /lib/package.lua:11: in field 'require'
[    0.40] [FAILED]     /PlotOS/bootInit.lua:447: in function </PlotOS/bootInit.lua:393>
[    0.40] [FAILED]     [C]: in function 'xpcall'
[    0.40] [FAILED]     machine:823: in local 'errorCapture'
[    0.40] [FAILED]     machine:833: in global 'xpcall'
[    0.40] [FAILED]     /PlotOS/bootInit.lua:507: in local 'res'
[    0.40] [FAILED]     /PlotOS/boot.lua:8: in main chunk
[    0.40] [FAILED]     init:15: in main chunk
[    0.40] [FAILED]     (...tail calls...)
[    0.40] [FAILED] Error loading file /lib/ipc.lua
[    0.40] [FAILED] Error: /PlotOS/bootInit.lua:249: /lib/process.lua:2: attempt to call a nil value (global 'require')
[    0.40] [FAILED] stack traceback:
[    0.40] [FAILED]     [C]: in function 'xpcall'
[    0.40] [FAILED]     machine:823: in upvalue 'errorCapture'
[    0.40] [FAILED]     machine:827: in global 'require'
[    0.40] [FAILED]     /lib/process.lua:2: in main chunk
[    0.40] [FAILED]     [C]: in function 'xpcall'
[    0.40] [FAILED]     machine:823: in local 'errorCapture'
[    0.40] [FAILED]     machine:833: in global 'xpcall'
[    0.40] [FAILED]     /PlotOS/bootInit.lua:241: in global 'raw_dofile'
[    0.40] [FAILED]     /lib/package.lua:11: in field 'require'
[    0.40] [FAILED]     /lib/ipc.lua:1: in main chunk
[    0.40] [FAILED]     ...
[    0.40] [FAILED]     machine:823: in local 'errorCapture'
[    0.40] [FAILED]     machine:833: in global 'xpcall'
[    0.40] [FAILED]     /PlotOS/bootInit.lua:241: in global 'raw_dofile'
[    0.40] [FAILED]     /lib/package.lua:11: in field 'require'
[    0.40] [FAILED]     /PlotOS/bootInit.lua:447: in function </PlotOS/bootInit.lua:393>
[    0.40] [FAILED]     [C]: in function 'xpcall'
[    0.40] [FAILED]     machine:823: in local 'errorCapture'
[    0.40] [FAILED]     machine:833: in global 'xpcall'
[    0.40] [FAILED]     /PlotOS/bootInit.lua:507: in local 'res'
[    0.40] [FAILED]     /PlotOS/boot.lua:8: in main chunk
[    0.40] [FAILED]     init:15: in main chunk
[    0.40] [FAILED]     (...tail calls...) is the error
[    0.40] [FAILED] stack traceback:
[    0.40] [FAILED]     [C]: in function 'xpcall'
[    0.40] [FAILED]     machine:823: in upvalue 'errorCapture'
[    0.45] [FAILED]     machine:827: in function <machine:823>
[    0.45] [FAILED]     [C]: in function 'error'
[    0.45] [FAILED]     /PlotOS/bootInit.lua:249: in global 'raw_dofile'
[    0.45] [FAILED]     /lib/package.lua:11: in field 'require'
[    0.45] [FAILED]     /lib/ipc.lua:1: in main chunk
[    0.45] [FAILED]     [C]: in function 'xpcall'
[    0.45] [FAILED]     machine:823: in local 'errorCapture'
[    0.45] [FAILED]     machine:833: in global 'xpcall'
[    0.45] [FAILED]     ...
[    0.45] [FAILED]     machine:823: in local 'errorCapture'
[    0.45] [FAILED]     machine:833: in global 'xpcall'
[    0.45] [FAILED]     /PlotOS/bootInit.lua:241: in global 'raw_dofile'
[    0.45] [FAILED]     /lib/package.lua:11: in field 'require'
[    0.45] [FAILED]     /PlotOS/bootInit.lua:447: in function </PlotOS/bootInit.lua:393>
[    0.45] [FAILED]     [C]: in function 'xpcall'
[    0.45] [FAILED]     machine:823: in local 'errorCapture'
[    0.45] [FAILED]     machine:833: in global 'xpcall'
[    0.45] [FAILED]     /PlotOS/bootInit.lua:507: in local 'res'
[    0.45] [FAILED]     /PlotOS/boot.lua:8: in main chunk
[    0.45] [FAILED]     init:15: in main chunk
[    0.45] [FAILED]     (...tail calls...)
[    0.45] [FAILED] Error loading file /lib/driver.lua
[    0.45] [FAILED] Error: /PlotOS/bootInit.lua:249: /PlotOS/bootInit.lua:249: /lib/process.lua:2: attempt to call a nil value (global 'require')
[    0.45] [FAILED] stack traceback:
[    0.45] [FAILED]     [C]: in function 'xpcall'
[    0.45] [FAILED]     machine:823: in upvalue 'errorCapture'
[    0.45] [FAILED]     machine:827: in global 'require'
[    0.45] [FAILED]     /lib/process.lua:2: in main chunk
[    0.45] [FAILED]     [C]: in function 'xpcall'
[    0.45] [FAILED]     machine:823: in local 'errorCapture'
[    0.45] [FAILED]     machine:833: in global 'xpcall'
[    0.45] [FAILED]     /PlotOS/bootInit.lua:241: in global 'raw_dofile'
[    0.45] [FAILED]     /lib/package.lua:11: in field 'require'
[    0.45] [FAILED]     /lib/ipc.lua:1: in main chunk
[    0.45] [FAILED]     ...
[    0.50] [FAILED]     machine:823: in local 'errorCapture'
[    0.50] [FAILED]     machine:833: in global 'xpcall'
[    0.50] [FAILED]     /PlotOS/bootInit.lua:241: in global 'raw_dofile'
[    0.50] [FAILED]     /lib/package.lua:11: in field 'require'
[    0.50] [FAILED]     /PlotOS/bootInit.lua:447: in function </PlotOS/bootInit.lua:393>
[    0.50] [FAILED]     [C]: in function 'xpcall'
[    0.50] [FAILED]     machine:823: in local 'errorCapture'
[    0.50] [FAILED]     machine:833: in global 'xpcall'
[    0.50] [FAILED]     /PlotOS/bootInit.lua:507: in local 'res'
[    0.50] [FAILED]     /PlotOS/boot.lua:8: in main chunk
[    0.50] [FAILED]     init:15: in main chunk
[    0.50] [FAILED]     (...tail calls...) is the error
[    0.50] [FAILED] stack traceback:
[    0.50] [FAILED]     [C]: in function 'xpcall'
[    0.50] [FAILED]     machine:823: in upvalue 'errorCapture'
[    0.50] [FAILED]     machine:827: in function <machine:823>
[    0.50] [FAILED]     [C]: in function 'error'
[    0.50] [FAILED]     /PlotOS/bootInit.lua:249: in global 'raw_dofile'
[    0.50] [FAILED]     /lib/package.lua:11: in field 'require'
[    0.50] [FAILED]     /lib/ipc.lua:1: in main chunk
[    0.50] [FAILED]     [C]: in function 'xpcall'
[    0.50] [FAILED]     machine:823: in local 'errorCapture'
[    0.50] [FAILED]     machine:833: in global 'xpcall'
[    0.50] [FAILED]     ...
[    0.50] [FAILED]     machine:823: in local 'errorCapture'
[    0.50] [FAILED]     machine:833: in global 'xpcall'
[    0.50] [FAILED]     /PlotOS/bootInit.lua:241: in global 'raw_dofile'
[    0.50] [FAILED]     /lib/package.lua:11: in field 'require'
[    0.50] [FAILED]     /PlotOS/bootInit.lua:447: in function </PlotOS/bootInit.lua:393>
[    0.50] [FAILED]     [C]: in function 'xpcall'
[    0.50] [FAILED]     machine:823: in local 'errorCapture'
[    0.50] [FAILED]     machine:833: in global 'xpcall'
[    0.50] [FAILED]     /PlotOS/bootInit.lua:507: in local 'res'
[    0.50] [FAILED]     /PlotOS/boot.lua:8: in main chunk
[    0.50] [FAILED]     init:15: in main chunk
[    0.55] [FAILED]     (...tail calls...) is the error
[    0.55] [FAILED] stack traceback:
[    0.55] [FAILED]     [C]: in function 'xpcall'
[    0.55] [FAILED]     machine:823: in upvalue 'errorCapture'
[    0.55] [FAILED]     machine:827: in function <machine:823>
[    0.55] [FAILED]     [C]: in function 'error'
[    0.55] [FAILED]     /PlotOS/bootInit.lua:249: in global 'raw_dofile'
[    0.55] [FAILED]     /lib/package.lua:11: in field 'require'
[    0.55] [FAILED]     /lib/driver.lua:4: in main chunk
[    0.55] [FAILED]     [C]: in function 'xpcall'
[    0.55] [FAILED]     machine:823: in local 'errorCapture'
[    0.55] [FAILED]     machine:833: in global 'xpcall'
[    0.55] [FAILED]     /PlotOS/bootInit.lua:241: in global 'raw_dofile'
[    0.55] [FAILED]     /lib/package.lua:11: in field 'require'
[    0.55] [FAILED]     /PlotOS/bootInit.lua:447: in function </PlotOS/bootInit.lua:393>
[    0.55] [FAILED]     [C]: in function 'xpcall'
[    0.55] [FAILED]     machine:823: in local 'errorCapture'
[    0.55] [FAILED]     machine:833: in global 'xpcall'
[    0.55] [FAILED]     /PlotOS/bootInit.lua:507: in local 'res'
[    0.55] [FAILED]     /PlotOS/boot.lua:8: in main chunk
[    0.55] [FAILED]     init:15: in main chunk
[    0.55] [FAILED]     (...tail calls...)
[    0.55] [FAILED] KERNEL PANIC
[    0.55] [FAILED] ----------------------------------------------------
[    0.55] [FAILED] Panic reason: Critical system failiure: /PlotOS/bootInit.lua:249: /PlotOS/bootInit.lua:249: /PlotOS/bootInit.lua:249: /lib/process.lua:2: attempt to call a nil value (global 'require')
[    0.55] [FAILED] stack traceback:
[    0.55] [FAILED]     [C]: in function 'xpcall'
[    0.55] [FAILED]     machine:823: in upvalue 'errorCapture'
[    0.55] [FAILED]     machine:827: in global 'require'
[    0.55] [FAILED]     /lib/process.lua:2: in main chunk
[    0.55] [FAILED]     [C]: in function 'xpcall'
[    0.55] [FAILED]     machine:823: in local 'errorCapture'
[    0.55] [FAILED]     machine:833: in global 'xpcall'
[    0.55] [FAILED]     /PlotOS/bootInit.lua:241: in global 'raw_dofile'
[    0.55] [FAILED]     /lib/package.lua:11: in field 'require'
[    0.60] [FAILED]     /lib/ipc.lua:1: in main chunk
[    0.60] [FAILED]     ...
[    0.60] [FAILED]     machine:823: in local 'errorCapture'
[    0.60] [FAILED]     machine:833: in global 'xpcall'
[    0.60] [FAILED]     /PlotOS/bootInit.lua:241: in global 'raw_dofile'
[    0.60] [FAILED]     /lib/package.lua:11: in field 'require'
[    0.60] [FAILED]     /PlotOS/bootInit.lua:447: in function </PlotOS/bootInit.lua:393>
[    0.60] [FAILED]     [C]: in function 'xpcall'
[    0.60] [FAILED]     machine:823: in local 'errorCapture'
[    0.60] [FAILED]     machine:833: in global 'xpcall'
[    0.60] [FAILED]     /PlotOS/bootInit.lua:507: in local 'res'
[    0.60] [FAILED]     /PlotOS/boot.lua:8: in main chunk
[    0.60] [FAILED]     init:15: in main chunk
[    0.60] [FAILED]     (...tail calls...) is the error
[    0.60] [FAILED] stack traceback:
[    0.60] [FAILED]     [C]: in function 'xpcall'
[    0.60] [FAILED]     machine:823: in upvalue 'errorCapture'
[    0.60] [FAILED]     machine:827: in function <machine:823>
[    0.60] [FAILED]     [C]: in function 'error'
[    0.60] [FAILED]     /PlotOS/bootInit.lua:249: in global 'raw_dofile'
[    0.60] [FAILED]     /lib/package.lua:11: in field 'require'
[    0.60] [FAILED]     /lib/ipc.lua:1: in main chunk
[    0.60] [FAILED]     [C]: in function 'xpcall'
[    0.60] [FAILED]     machine:823: in local 'errorCapture'
[    0.60] [FAILED]     machine:833: in global 'xpcall'
[    0.60] [FAILED]     ...
[    0.60] [FAILED]     machine:823: in local 'errorCapture'
[    0.60] [FAILED]     machine:833: in global 'xpcall'
[    0.60] [FAILED]     /PlotOS/bootInit.lua:241: in global 'raw_dofile'
[    0.60] [FAILED]     /lib/package.lua:11: in field 'require'
[    0.60] [FAILED]     /PlotOS/bootInit.lua:447: in function </PlotOS/bootInit.lua:393>
[    0.60] [FAILED]     [C]: in function 'xpcall'
[    0.60] [FAILED]     machine:823: in local 'errorCapture'
[    0.60] [FAILED]     machine:833: in global 'xpcall'
[    0.60] [FAILED]     /PlotOS/bootInit.lua:507: in local 'res'
[    0.65] [FAILED]     /PlotOS/boot.lua:8: in main chunk
[    0.65] [FAILED]     init:15: in main chunk
[    0.65] [FAILED]     (...tail calls...) is the error
[    0.65] [FAILED] stack traceback:
[    0.65] [FAILED]     [C]: in function 'xpcall'
[    0.65] [FAILED]     machine:823: in upvalue 'errorCapture'
[    0.65] [FAILED]     machine:827: in function <machine:823>
[    0.65] [FAILED]     [C]: in function 'error'
[    0.65] [FAILED]     /PlotOS/bootInit.lua:249: in global 'raw_dofile'
[    0.65] [FAILED]     /lib/package.lua:11: in field 'require'
[    0.65] [FAILED]     /lib/driver.lua:4: in main chunk
[    0.65] [FAILED]     [C]: in function 'xpcall'
[    0.65] [FAILED]     machine:823: in local 'errorCapture'
[    0.65] [FAILED]     machine:833: in global 'xpcall'
[    0.65] [FAILED]     /PlotOS/bootInit.lua:241: in global 'raw_dofile'
[    0.65] [FAILED]     /lib/package.lua:11: in field 'require'
[    0.65] [FAILED]     /PlotOS/bootInit.lua:447: in function </PlotOS/bootInit.lua:393>
[    0.65] [FAILED]     [C]: in function 'xpcall'
[    0.65] [FAILED]     machine:823: in local 'errorCapture'
[    0.65] [FAILED]     machine:833: in global 'xpcall'
[    0.65] [FAILED]     /PlotOS/bootInit.lua:507: in local 'res'
[    0.65] [FAILED]     /PlotOS/boot.lua:8: in main chunk
[    0.65] [FAILED]     init:15: in main chunk
[    0.65] [FAILED]     (...tail calls...) is the error
[    0.65] [FAILED] stack traceback:
[    0.65] [FAILED]     [C]: in function 'xpcall'
[    0.65] [FAILED]     machine:823: in upvalue 'errorCapture'
[    0.65] [FAILED]     machine:827: in function <machine:823>
[    0.65] [FAILED]     [C]: in function 'error'
[    0.65] [FAILED]     /PlotOS/bootInit.lua:249: in global 'raw_dofile'
[    0.65] [FAILED]     /lib/package.lua:11: in field 'require'
[    0.65] [FAILED]     /PlotOS/bootInit.lua:447: in function </PlotOS/bootInit.lua:393>
[    0.65] [FAILED]     [C]: in function 'xpcall'
[    0.65] [FAILED]     machine:823: in local 'errorCapture'
[    0.65] [FAILED]     machine:833: in global 'xpcall'
[    0.70] [FAILED]     /PlotOS/bootInit.lua:507: in local 'res'
[    0.70] [FAILED]     /PlotOS/boot.lua:8: in main chunk
[    0.70] [FAILED]     init:15: in main chunk
[    0.70] [FAILED]     (...tail calls...)
[    0.70] [FAILED] ----------------------------------------------------
[    0.70] [FAILED] Stack trace:
[    0.70] [FAILED] stack traceback:
[    0.70] [FAILED]     /PlotOS/bootInit.lua:512: in local 'res'
[    0.70] [FAILED]     /PlotOS/boot.lua:8: in main chunk
[    0.70] [FAILED]     init:15: in main chunk
[    0.70] [FAILED]     (...tail calls...)
[    0.70] [FAILED] ----------------------------------------------------
[    0.70] [FAILED] System Information:
[    0.70] [FAILED] OS: PlotOS 0.0.4-alpha
[    0.70] [FAILED] Status: 0, Uptime: 3.30s
[    0.70] [FAILED] Boot Address: fc2c9930-a622-4204-9848-1d81f24b673d
[    0.70] [FAILED] Machine Address: 84a0f153-c6f4-4701-a633-eed5bccfd1de
[    0.70] [FAILED] Hardware Information:
[    0.70] [FAILED] Architecture: Lua 5.3
[    0.70] [FAILED] Memory: 2048KB total, 1584KB free
[    0.70] [FAILED] GPU Resolution: 160x50
[    0.70] [FAILED] Connected Components:
[    0.70] [FAILED] eeprom: 967f243f-3126-41ff-b311-54912f937a36
[    0.70] [FAILED] filesystem: fb705fb0-0e15-439b-9f9c-12e8443973f8
[    0.70] [FAILED] drive: 714fde65-5f5f-455a-830c-0f035c8928c3
[    0.70] [FAILED] computer: 84a0f153-c6f4-4701-a633-eed5bccfd1de
[    0.70] [FAILED] ocelot: 3e53b338-34fb-4cac-b9ad-2b990f0a499b
[    0.70] [FAILED] keyboard: 890d508a-03e2-4a2c-8af2-eae287b5556f
[    0.70] [FAILED] filesystem: fc2c9930-a622-4204-9848-1d81f24b673d
[    0.70] [FAILED] screen: ce258025-98c9-4fe2-b77f-ea6c45a163c0
[    0.70] [FAILED] gpu: d4400ce2-3670-4de9-bf7f-9aa3b1afaa73
[    0.70] [FAILED] Environment State:
[    0.70] [FAILED] LOW_MEM: false, VERY_LOW_MEM: false
